.chloggen/deprecated-http.client_ip.yaml

```
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'deprecation'
```

(since not a new deprecation, just documenting an old deprecation)
```suggestion
change_type: 'bug_fix'
```

---

I originally had bug fix but changed to deprecation on the advice of a different reviewer https://github.com/open-telemetry/semantic-conventions/pull/1025#discussion_r1596555972